src/app/modules/group/components/suggestion-of-activities/suggestion-of-activities.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container *ngIf="state$ | async as state">
+  <div class="spinner-container" *ngIf="state.isFetching">
+    <p-progressSpinner [style]="{width: '24px', height: '24px'}"></p-progressSpinner>
+  </div>
+  <div *ngIf="state.isError" i18n>Error while loading the activities</div>
+  <div *ngIf="state.isReady && state.data.length === 0" i18n>
+    There are no activities
+  </div>
+  <p-table
+      class="alg-table --overlay-panel"
+      [value]="state.data"
+      *ngIf="state.isReady && state.data.length > 0"
+  >
+    <ng-template pTemplate="body" let-item>
+      <tr>
+        <td><span class="type">{{ item.activity.type }}</span></td>
+        <td>
+          <a class="alg-link dots" [routerLink]="item.activity | rawItemLink">
```

Comment:
A small improvement... but you'll need my newly merged branch (so to rebase master).
Actually, here you know more that just the id of the item. As the item is a root, you know that its path is `[]`, this way you'll not have to call the service to find the path. 
So either you build a `RouteItem` and convert it to a link before passing it to the template, or you adapt your pipe to give the path... but in this case it is not a rawItemLink anymore...

... ok merge it as it and I'll do the improvement myself. I will be easier. :-)